tests/entry_page_test.js

```diff
@@ -2,6 +2,45 @@ const utils = require('../utils/utils');
 
 Feature('entry_page');
 
+const logExperimentDetails = (
+    countOfActiveKeys, countOfEntryPoints, allActiveKeysEntryPointInfo, countOfConfirmations, confirmations
+) => {
+    console.log(`Active keys: ${countOfActiveKeys}`);
+    console.log(`Entry points: ${countOfEntryPoints}`);
+    allActiveKeysEntryPointInfo.forEach(key => {
+        console.log(`${key.id} | isEntryPoint: ${key.isEntryPoint}`);
+    })
+    console.log(`Confirmations: ${countOfConfirmations}`);
+    console.log(confirmations);
+}
+
+const validateEntryPoints = (allActiveKeysEntryPointInfo, entryPointValue) => {
+    let isValid;
+    allActiveKeysEntryPointInfo.forEach(key => {
+        if (key.isEntryPoint === entryPointValue) {
+            isValid === false ? isValid = false : isValid = true;
+        } else {
+            isValid = false;
+        }
+    })
+    return isValid;
+}
+
+const checkOnlyEntryPointConfirmations = (confirmations, allActiveKeysEntryPointInfo) => {
+    let validChecks = 0;
+    confirmations.forEach(confirmation => {
+        allActiveKeysEntryPointInfo.find(key => {
+            if (key.isEntryPoint) {
+                confirmation.indexOf(key.id) > -1 ? validChecks += 1 : validChecks -= 1;
```

Comment:
An early return here if the check failed would also reduce the amount of thinking needed to understand the test.

Even if it would increase the code -- there are 2 specific things being checked here, so making that clear would help